assets/pages/paper-subscription-landing/components/content/content.jsx

```diff
@@ -0,0 +1,179 @@
+      <UnorderedList items={[
+        'Your paper will arrive before 7am from Monday to Saturday and before 8.30am on Sunday',
+        'We can’t delivery to individual flats, or apartments within blocks because we need access to your post box to deliver your paper',
+        'You can pause your subscription for up to 36 days a year. So if you’re going away anywhere, you won’t have to pay for the papers that you miss',
+        ]}
+      />
+    </ProductPageTextBlock>
+  </ProductPageContentBlock>
+
+);
+
+function trackedLink(href: string, text: string, onClick: Function) {
+  return <a href={href} onClick={onClick}>{text}</a>;
+}
+
+
+// ----- Render ----- //
+
+type PropTypes = {|
+  selectedTab: ActiveTabState,
+  setTabAction: typeof setTab
+|};
+
+class Content extends Component<PropTypes> {
+
+  componentDidUpdate({ selectedTab }) {
+    if (selectedTab !== this.props.selectedTab && this.tabRef) {
+      // $FlowIgnore
+      this.tabRef.focus({
+        preventScroll: true,
+      });
+    }
+  }
+
+  tabRef: ?HTMLDivElement;
+
+  render() {
+    const { selectedTab, setTabAction } = this.props;
+
+    const collectionPage = (
+      <div className="paper-subscription-landing-content__focusable" tabIndex={-1} ref={(d) => { this.tabRef = d; }}>
+        <ContentVoucherFaqBlock />
+        <ContentForm selectedTab={selectedTab} setTabAction={setTabAction} title="Pick your subscription package below: Voucher" />
+        <ContentHelpBlock
+          faqLink={trackedLink(
+            'https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions',
+            'Subscriptions FAQs',
+            sendClickedEvent('paper_subscription_collection_page-subscription_faq_link'),
+          )}
+          telephoneLink={trackedLink(
+            'tel:+4403303336767',
+            '0330 333 6767',
+            sendClickedEvent('paper_subscription_collection_page-telephone_link'),
+          )}
+        />
+      </div>
+    );
+
+    const deliveryPage = (
+      <div className="paper-subscription-landing-content__focusable" tabIndex={-1} ref={(d) => { this.tabRef = d; }}>
+        <ContentDeliveryFaqBlock setTabAction={setTabAction} />
+        <ContentForm selectedTab={selectedTab} setTabAction={setTabAction} title="Pick your subscription package below: Delivery" />
+        <ContentHelpBlock
+          faqLink={trackedLink(
+            'https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions',
+            'Subscriptions FAQs',
+            sendClickedEvent('paper_subscription_delivery_page-subscription_faq_link'),
+          )}
+          telephoneLink={trackedLink(
+            'tel:+4403303336767', // yes, we're using a phone number as a link
+            '0330 333 6767',
+            sendClickedEvent('paper_subscription_delivery_page-telephone_link'),
+          )}
+        />
+      </div>
+    );
+
+    return selectedTab === 'collection' ? (collectionPage) :
```

Comment:
If you changed these from `const`s to function calls or separate components we would only instantiate the one that is used.

Reply:
I was scared to do this initially because of the refs but i'll do it, will make a nicer structure